tests/acceptance/25_cf-execd/mail_exclude_filter.cf

```diff
@@ -0,0 +1,38 @@
+# Tests that exclude mail filters, causes everything but those lines to be
+# mailed by cf-execd.
```

Comment:
The description is hard to understand. Maybe something like "Tests that exclude mail filter causes excluded lines are not mailed by cf-execd.".